### PR TITLE
Feature: Document Type Structure - configure collection UI

### PR DESCRIPTION
**Pull request by leekelleher**
Adds a new component `umb-input-collection-configuration` for picking an existing collection configuration (data-type), or to create a new one.

@nielslyngsoe may want to review my usage of the `UMB_DATA_TYPE_PICKER_FLOW_DATA_TYPE_PICKER_MODAL`, _(as the naming may feel wrong in this scenario)_.

The rest is to wire up the UI with the server calls to save the data.

### CODE REVIEW

File: src/packages/documents/document-types/workspace/views/structure/document-type-workspace-view-structure.element.ts [hunk 1]
```diff
@@ -77,9 +99,20 @@ export class UmbDocumentTypeWorkspaceViewStructureElement extends UmbLitElement
 				</umb-property-layout>
 			</uui-box>
 			<uui-box headline="Presentation">
-				<umb-property-layout alias="Root" label="Collection view">
-					<div slot="description">Provides an overview of child content and hides it in the tree.</div>
-					<div slot="editor"><uui-toggle label="Display children in a Collection view"></uui-toggle></div>
+				<umb-property-layout alias="collection" label="Collection">
+					<div slot="description">
+						Configures the content item to show list of its children, the children will not be shown in the tree.
```
Comment (iOvergaard) [inline src/packages/documents/document-types/workspace/views/structure/document-type-workspace-view-structure.element.ts:104]: Would be good to include the localization key for this

File: src/packages/documents/document-types/workspace/views/structure/document-type-workspace-view-structure.element.ts [hunk 2]
```diff
@@ -77,9 +99,20 @@ export class UmbDocumentTypeWorkspaceViewStructureElement extends UmbLitElement
 				</umb-property-layout>
 			</uui-box>
 			<uui-box headline="Presentation">
```
Comment (iOvergaard) [inline src/packages/documents/document-types/workspace/views/structure/document-type-workspace-view-structure.element.ts:101]: Not part of this PR, but could you add some localization here and maybe in the file in general?

Comment (leekelleher) [reply src/packages/documents/document-types/workspace/views/structure/document-type-workspace-view-structure.element.ts:101]: Should I do as part of this PR, or separate PR? Just thinking whether we want the feature in preview008 or for beta001?

Comment (iOvergaard) [reply src/packages/documents/document-types/workspace/views/structure/document-type-workspace-view-structure.element.ts:101]: New PR is fine 👍

File: src/packages/documents/document-types/workspace/views/structure/document-type-workspace-view-structure.element.ts (hunk 1 above)
Comment (leekelleher) [reply src/packages/documents/document-types/workspace/views/structure/document-type-workspace-view-structure.element.ts:104]: @iOvergaard I'll update this PR with the localization keys that are related to this PR and do the separate PR for the rest of the Document Type Structure view.
